include/crm/crm.h

```diff
@@ -66,7 +66,7 @@ extern "C" {
  * >=3.0.13: Fail counts include operation name and interval
  * >=3.2.0:  DC supports PCMK_EXEC_INVALID and PCMK_EXEC_NOT_CONNECTED
  */
-#  define CRM_FEATURE_SET		"3.17.1"
+#  define CRM_FEATURE_SET		"3.17.2"
```

Comment:
I just realized we forgot to do this for cibadmin, but it's fine for this bump to cover both

Reply:
`cibadmin` doesn't use formatted output yet. The user isn't aware of GOption.

Reply:
Oh right, juggling too many PRs in my head :)